app/controllers/success_controller.rb

```diff
@@ -12,14 +12,22 @@ def next_path
   private
 
   def before_rendering_edit_hook
-    Export.create_and_enqueue(destination: :fax,
-                              snap_application: current_snap_application)
+    Export.create_and_enqueue(
+      destination: :fax,
+      snap_application: current_snap_application,
+    )
+
+    ApplicationSubmittedSmsJob.perform_later(
+      snap_application: current_snap_application,
+    )
```

Comment:
If we add an `:sms` destination in the `Export` model, it will automatically track when and how we send SMSs.

Reply:
yeah I considered doing that, but felt like `export` was specific to exporting a PDF, conceptually, rather than "sending out a thing" -- what do you think?

Reply:
Maybe it means we need a rename to `Communique`? It's more about tracking when and where we communicate outwards (either via MiBridges, Fax, Email, SMS, etc) so that an admin can drill into the time we send stuff?